inputs/templates/terra_workspaces/cohort_mode/workflow_configurations/MainVcfQc.json.tmpl

```diff
@@ -10,6 +10,7 @@
 
   "MainVcfQc.sv_pipeline_docker": "${workspace.sv_pipeline_docker}",
   "MainVcfQc.sv_base_mini_docker": "${workspace.sv_base_mini_docker}",
+  "MainVcfQc.samtools_cloud_docker": "${workspace.samtools_cloud_docker}",
```

Comment:
Does this input also need to be added to MakeCohortVcf, the Batch WDL, and the single-sample WDL? Or maybe just use sv_pipeline_docker to avoid having to do that?

Reply:
We could use anything derived from `samtools_cloud_docker`, so `sv_pipeline_docker` would work. I went with `samtools_cloud_docker` because it's smaller, but I don't think that's a huge deal. I think the best way forward would be to keep `samtools_cloud_docker` as the input for the Task in `Utils.wdl`, but then just pass `sv_pipeline_docker` as the corresponding input from `MainVcfQc.wdl`. I'll make that change unless you object.

Reply:
Sounds like a good plan to me

Reply:
I followed this plan. However it's now moot: the task for renaming VCF samples is not called. I'm inclined to delete it, since it is not used and doesn't use the preferred TSV input format. However, I figured I would wait for reviewer confirmation.

Reply:
Yes, please go ahead and delete this input as well as the old task